src/app/core/services/personal-cards.service.spec.ts

```
@@ -1537,12 +1537,46 @@
+
+      const result = personalCardsService.htmlFormUrl(baseUrl, accessToken, isMfaFlow);
+      const expectedExtraParams = 'configName=Aggregation&callback=https://www.fylehq.com';
+      const expectedHtml = `<form id="fastlink-form" name="fastlink-form" action="${baseUrl}" method="POST">
+                          <input name="accessToken" value="Bearer ${accessToken}" hidden="true" />
+                          <input  name="extraParams" value="${expectedExtraParams}" hidden="true" />
+                          </form> 
+                          <script type="text/javascript">
+                          document.getElementById("fastlink-form").submit();
+                          </script>
+                          `;
+      const expectedEncodedHtml = 'data:text/html;base64,' + btoa(expectedHtml);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)